app/assets/stylesheets/_github-ribbon.css.sass

```diff
@@ -0,0 +1,48 @@
+// Responsive GitHub Ribbon
+
+.ribbon
+  background-color: #7d7d7d
+  overflow: hidden
```

Comment:
Problem was also that the header was set to less then 100%, so I put the ribbon in the body. Thanks for the tip though!